### DISCUSSION TITLE
1.5.0.2

### DISCUSSION THREAD

**Discussion by noberumotto** [Planshit/Tai#277] [General]
> 优化了统计时长模块以提升其准确性，全新的浏览器访问统计功能...

### 主要更新
1. 优化了统计时长的代码，使统计软件应用时长更加准确；
2. 全新的浏览器访问统计功能，现在需要安装浏览器拓展才能正常统计。目前只支持能够安装Chrome拓展的浏览器，在浏览器中打开拓展管理>打开 `开发模式` >加载已解压拓展>选择解压后的Tai浏览器拓展文件夹即可完成安装。记得还是需要在Tai的设置中启用浏览器统计功能。Tai浏览器拓展与Tai打包在同一个压缩包，名称为：Tai chrome extension.zip。

### 其他
1. 修复了一些问题；
2. 新增了 `启动软件时显示主界面` 设置选项，在 `常规` 页中；
3. 新增了 `睡眠监测` 设置选项，可以手动关闭睡眠监测功能，以能在离开电脑时也继续统计时长，一般情况不建议关闭。在 `行为` 页中；
4. 现在开机自启时不会再显示主界面了；（@Steesha）

<hr /><em>This discussion was created from the release <a href='https://github.com/Planshit/Tai/releases/tag/1.5.0.2'>1.5.0.2</a>.</em>

**Comment by noberumotto** [comment]
预览版，有问题请在此回复~

**Reply by Erick200404** [reply]
请问可以对网站进行分类吗
